### DISCUSSION TITLE
mutateRelationshipDataBeforeFillUsing Not Working

### DISCUSSION THREAD

**Discussion by PegeDev** [filamentphp/filament#14598] [Help]
### Package

Form builder

### Package Version

v3.2.109

### How can we help you?

I use mutateRelationshipDataBeforeFillUsing in Group layout in Form builder, the data I change in the mutation is not filled in the input.. can you help me with this problem? I have been looking for an answer for quite a long time about this.. but there is no response at all. Here are some photos I attach

![image](https://github.com/user-attachments/assets/9603c4b6-36ec-4327-9484-f0cf3a904359)
![image](https://github.com/user-attachments/assets/bc1cf2f5-84b2-487f-86b2-82fef3aa2603)

**Comment by leandrocfe** [comment]
Could you share the fields?

**Reply by leandrocfe** [reply]
You could also try

```php
Radio::make('isTanah')
    ->options([
        '1' => 'Tidak Ada',
        '2' => 'Ada',
    ])
    ->formatStateUsing(fn (?string $state) => $state ?? '1')
```
